sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,10 +2,18 @@
 
 ## 7.1.1 (Unreleased)
```

Comment:
should we add the date here or wait on that until tomorrow?

Reply:
will merge first and resolve in your PR :P